macros/trace_object/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "mmtk-macro-trace-object"
+version = "0.1.0"
```

Comment:
We should give it the same version as mmtk-core's

Reply:
Done.